front/src/modules/scenario/components/ScenarioExplorer/ScenarioExplorer.tsx

```diff
@@ -44,8 +44,8 @@ const ScenarioExplorer = ({
       { skip: !globalProjectId && !globalStudyId }
     );
 
-  const { currentData: scenarioV2, isSuccess: isScenarioSuccess } =
-    osrdEditoastApi.endpoints.getV2ProjectsByProjectIdStudiesAndStudyIdScenariosScenarioId.useQuery(
+  const { currentData: Scenario, isSuccess: isScenarioSuccess } =
```

Comment:
We don't use capital case for consts (it's not used as a type here)

```suggestion
  const { currentData: scenario, isSuccess: isScenarioSuccess } =
```